### DISCUSSION TITLE
"Jupyter notebook failed to launch. {0}" - I cannot run Python scripts with the extension in Code - OSS

### DISCUSSION THREAD

**Discussion by ew1Rvf24** [microsoft/vscode-jupyter#15083] [Questions and Answers]
![Screenshot_2024-01-27-12-24-46-024_com termux x11](https://github.com/microsoft/vscode-jupyter/assets/153981553/1f84ed65-9196-4e45-b376-a6d3054357e7)

### What happened?
I'm using the Code-OSS Jupyter extension in a Termux-X11 environment. The extension can connect to a pre-launched ipykernel, but always fails to start kernel when just using local Python environments. Reverting the extension version to earlier builds doesn't help to solve the problem. I'll be grateful if anyone can help me to make it work. 🙏

### Code - OSS Version: 
Version: 1.85.2 Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681 Date: 2024-01-18T19:46:30.066Z
Electron: 25.9.7 ElectronBuildId: undefined Chromium: 114.0.5735.289 Node.js: 18.15.0 V8: 11.4.183.29-electron.0
OS: Linux arm64 5.10.136-android12-9-00021-g821df8f5bd36-ab9585204

### Jupyter Extension Version
v2023.9.100

### Jupyter logs
[2-Jupyter.log](https://github.com/microsoft/vscode-jupyter/files/14071004/2-Jupyter.log)

### Coding Language and Runtime Version
Python 3.11.6

### Running Jupyter locally or remotely?
Local

**Comment by DonJayamanne** [comment]
That's an old version of the jupyter extension
Please update to the latest version

**Reply by ew1Rvf24** [reply]
Thanks a lot! But the problem is still there after I tried to update to the latest build. Here's the recent log file.
[1-Jupyter.log](https://github.com/microsoft/vscode-jupyter/files/14072326/1-Jupyter.log)

Is there something I can do through settings.json or else?

**Reply by DonJayamanne** [reply]
@ew1Rvf24 
> dlopen failed: library "libstdc++.so.6" not found
looks like you have some dependency issue, 
YOu might want to try and install libstdc, see here https://stackoverflow.com/questions/19386651/how-to-fix-usr-lib-libstdc-so-6-version-glibcxx-3-4-15-not-found
